scripts/diagnostics.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+## Script which contains the diagnostics we wish to generate.
+## This script will output all diagnostics to stdout.
+
+# Start by printing the date
+date
+
+write_to_stdout() {
+  echo -e "$@"
+}
+
+file_born_date() {
+  if [ -f $1 ]; then
+    if [ `uname` == "Darwin" ]; then
+      echo `stat -f "%Sm" -t "%Y-%m-%d %H:%M" "$1"`
+    else
+      echo `date -d @$(stat -c %Y "$1") +%Y-%m-%d\ %H:%M`
+    fi
+  fi
+}
+
```

Comment:
file_born_date is not used in this script... but _may_ be useful in the future...